internal/adminapi/endpoints.go

```diff
@@ -174,7 +174,7 @@ func adminAPIFromEndpoint(
 	case cfgtypes.ServiceScopedPodDNSStrategy:
 		if service.Name == "" {
 			return DiscoveredAdminAPI{}, fmt.Errorf(
-				"service name is empty for an endpoint with TargetRef %s/%s",
+				"Service name is empty for an endpoint with TargetRef %s/%s",
```

Comment:
This is just an `error` and according to Google's style guide 

https://google.github.io/styleguide/go/decisions.html#:~:text=Error%20strings%20should%20not%20be,being%20printed%20to%20the%20user.

we shouldn't capitalise the first letter here.

```suggestion
				"service name is empty for an endpoint with TargetRef %s/%s",
```

Reply:
oh Thanks!! Got it